core/src/main/scala/scalaz/Tannen.scala

```diff
@@ -0,0 +1,23 @@
+package scalaz
+
+/** Composes a Functor on the outside of a Bifunctor.
+  *
+  * [[https://hackage.haskell.org/package/bifunctors-5.5/docs/Data-Bifunctor-Tannen.html#t:Tannen]]
+  */
+final case class Tannen[F[_], G[_, _], A, B](f: F[G[A, B]])
+
+object Tannen extends TannenInstances
+
+sealed abstract class TannenInstances {
+  implicit def TannenFunctor[F[_], G[_, _], E](implicit F: Functor[F], G: Functor[G[E, ?]]): Functor[Tannen[F, G, E, ?]] =
```

Comment:
For some reason, bifunctors has this as:

```haskell
(Functor f, Bifunctor p) => Functor (Tannen f p a)
```

I've relaxed Bifunctor to Functor. This adds a flexible context, but I don't think that'd be a problem. Is there a reason why bifunctors does what it does?

Reply:
I changed to what bifunctors does because otherwise the instance is the same as:

```haskell
(Functor f, Functor g) => Functor (Compose f g)
```